config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -65,6 +65,64 @@ presubmits:
             memory: 14Gi
         securityContext:
           privileged: true
+
+  - name: pull-cos-containerd-e2e-ubuntu-gce
+    always_run: false
+    cluster: k8s-infra-prow-build
+    skip_branches:
+    - release-\d+\.\d+ # per-release image
+    annotations:
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
```

Comment:
Do we need preset-e2e-cos-containerd in here instead or in addition to these? Not sure what that preset does but it is on the ci job

Reply:
that adds the `--env=LOG_DUMP_SYSTEMD_SERVICES=containerd` which i already incorporated.